ServiceLayer/Sources/ServiceLayer/Utilities/AppPreferences.swift

```diff
@@ -91,6 +98,18 @@ public extension AppPreferences {
         set { self[.animateAvatars] = newValue.rawValue }
     }
 
+    var keyboardType: KeyboardType {
```

Comment:
please add inline comment describing how this var is used.

Reply:
Thanks. This would be the only var in that file with such an inline comment, though. That seems very inconsistent to me.

Reply:
Yes you are right, I changed my mind on this